platform/broadcom/sonic-platform-modules-tencent/common/sonic_platform/chassis.py

```diff
@@ -0,0 +1,578 @@
+#!/usr/bin/env python
```

Comment:
This file is a copy of [https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/chassis_base.py](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/chassis_base.py). Please reuse instead of copying